templates/debug/source_code.html.twig

```diff
@@ -30,15 +30,15 @@
                     <div class="tab-content" id="myTabContent">
                         <div class="tab-pane show active" id="controller-code" role="tabpanel" aria-labelledby="controller-tab">
                             {% if controller %}
-                                <p class="file-link">{{ controller.file_path|format_file(controller.starting_line) }}</p>
+                                <p class="file-link">{{ link_source_file(controller.file_path, controller.starting_line) }}</p>
```

Comment:
why making it a Twig function and not a filter ?

Reply:
Because both arguments are equally required. It seems more coherent to me to have them at the same level.